cumulus_library/.sqlfluff

```diff
@@ -3,7 +3,7 @@ templater = jinja
 dialect = athena
 sql_file_exts = .sql,.sql.jinja
 # this rule overfires on athena nested arrays
-exclude_rules=references.from,structure.column_order
+exclude_rules=references.from,structure.column_order,aliasing.unused
```

Comment:
The AL05 rule is so over-agressive that, rather than continuing to ask users to remember to apply QA rules for it, I'm disabling it for now. This means that a query like this will no longer trigger a correction:

```
SELECT t2.row2
FROM table,
unnest (table.array) as t1 (row1) <--- this would trigger this rule even though it's valid
unnest (t1.row1.array) as t2 (row2)
```

I'm watching a couple issues on the sqlfluff repository related to this.